### DISCUSSION TITLE
v2.10.1

### DISCUSSION THREAD

**Discussion by tomkerkhove** [kedacore/charts#408] [Announcements]
Chart: v2.10.1 | App: v2.10.0

## TL;DR

```shell
helm repo add kedacore https://kedacore.github.io/charts
helm repo update
helm install keda kedacore/keda --version 2.10.1 -n keda --create-namespace
```

## What is new?

- Provide support for adding `podLabels` override in the webhook deployment (#404- Contributed by @pari 🎉)
- Include minutes and seconds to cert-manager certificates (#402- Contributed by @JorTurFer 🎉)

### Fixes / Changes

- Fixed an bug where  `POD_NAMESPACE` environment variable was not added to deployments (#405 - Contributed by @michemache 🎉)
- Enforce Kubernetes v1.24 or above to align with [KEDA's cluster compatibility](https://keda.sh/docs/2.10/operate/cluster/#kubernetes-compatibility).

### Breaking Changes

None.

## New Contributors
* @pari- made their first contribution in https://github.com/kedacore/charts/pull/404
* @michemache made their first contribution in https://github.com/kedacore/charts/pull/405

**Full Changelog**: https://github.com/kedacore/charts/compare/v2.10.0...v2.10.1

<hr /><em>This discussion was created from the release <a href='https://github.com/kedacore/charts/releases/tag/v2.10.1'>v2.10.1</a>.</em>
